# Is this month's key phrase "Medium-Intensity Residential"?
> Recent sale prices and assessments by land use designation

- author: Josh Carp
- categories: [land-use]

In [1]:
#hide

# Hack: Ignore shapely/geos version mismatch warning
import warnings
warnings.simplefilter("ignore")

import pandas as pd
from google.cloud import bigquery

bq = bigquery.Client()

pd.set_option("display.max_rows", 500)

## Property transactions in Charlottesville: Hot market ~~for parcels designated as "Middle Intensity" in Future Land Use Plan~~

A response to [March 2022 property transactions in Charlottesville: Hot market for parcels designated as "Middle Intensity" in Future Land Use Plan](https://communityengagement.substack.com/p/march-2022-property-transactions)

I always make time to read Sean Tubbs's [newsletter](https://communityengagement.substack.com/), and I was interested to see what he has to say about the new "Medium-Intensity Residential" land use designation adopted in last year's [Comprehensive Plan](https://www.charlottesville.gov/DocumentCenter/View/7073/Comprehensive-Plan-Document---2021-1115-Final?bidId=). Sean noticed a number of properties with this designation, or located close to properties with this designation, that sold for high prices, some far above their most recent assessments. He [wrote](https://twitter.com/cvilletowncrier/status/1512836544486387720) that

> This month's key phrase is "Medium Intensity Residential".

and noted that

> ...properties with the Medium Intensity Residential designation are selling well over the 2022 assessment.

Sean writes that "I intend no analysis". Fair enough, but let's do some analysis! The city provides mostly current data on assessments, property transactions, and parcel details at its [Open Data Portal](https://opendata.charlottesville.org/), which I have been mirroring to a public dataset using [Google BigQuery](https://console.cloud.google.com/bigquery?dataset=cville_open_data&p=cvilledata&d=cville_open_data&page=dataset) for ease of use. Let's dig into the claims in the blog post. As I read it, there are two key arguments that we can evaluate:

- Parcels with the Medium-Intensity Residential designation have been selling for over their official assessments
- The difference between sale prices and assessments is greater for Medium-Intensity Residential than for other designations

First, let's compare median sale prices, assessments, and the ratio between the two across land use designations. I'm looking at all sales in 2022 to begin with. Note that I'm excluding sales with a price of $0 and sales that included multiple parcels.

In [2]:
#collapse-hide
bq.query(
    """
    select distinct
      percentile_cont(sales.saleamount, 0.5) over (partition by flum.desig_oct) as saleamount,
      percentile_cont(assessments.totalvalue, 0.5) over (partition by flum.desig_oct) as assessment,
      percentile_cont(sales.saleamount / assessments.totalvalue, 0.5) over (partition by flum.desig_oct) as saleassessmentratio,
      count(*) over (partition by flum.desig_oct) as count,
      flum.desig_oct as designation,
    from `cvilledata.cville_open_data.real_estate_sales` sales
    join `cvilledata.cville_open_data.real_estate_all_assessments` assessments on sales.parcelnumb = assessments.parcelnumb and assessments.taxyear = 2022
    join `cvilledata.cville_plans_together.flum_202110` flum on sales.parcelnumb = flum.parcelnumb
    where sales.saledate >= '2022-01-01'
      and sales.saleamount > 0
      and sales.bookpage not in (
        select
          bookpage
        from `cvilledata.cville_open_data.real_estate_sales`
        where saledate >= '2022-01-01'
          and saleamount > 0
        group by bookpage
        having count(*) > 1
      )
      and flum.desig_oct in (
        'General Residential',
        'Medium-Intensity Residential',
        'High-Intensity Residential'
      )
    order by count desc
    """
).result().to_dataframe()

saleamount  assessment  saleassessmentratio  count  \
0    423750.0    345100.0             1.128854    100   
1    355350.0    364200.0             1.054195     24   
2    430000.0    440800.0             0.975198      2   

                    designation  
0           General Residential  
1  Medium-Intensity Residential  
2    High-Intensity Residential

As Sean noted, Medium-Intensity Residential parcels are selling for over their assessments, with the median ratio of sale price to assessed value at 1.05, meaning that sales were typically about 5% over assessments. However, General Residential parcels had an even higher ratio of 1.13, with the median parcel selling for about 13% more than its assessed value. But Sean only considered March; let's use that same filter to make sure we're looking at the same parcels:

In [3]:
#collapse-hide
bq.query(
    """
    select distinct
      percentile_cont(sales.saleamount, 0.5) over (partition by flum.desig_oct) as saleamount,
      percentile_cont(assessments.totalvalue, 0.5) over (partition by flum.desig_oct) as assessment,
      percentile_cont(sales.saleamount / assessments.totalvalue, 0.5) over (partition by flum.desig_oct) as saleassessmentratio,
      count(*) over (partition by flum.desig_oct) as count,
      flum.desig_oct as designation,
    from `cvilledata.cville_open_data.real_estate_sales` sales
    join `cvilledata.cville_open_data.real_estate_all_assessments` assessments on sales.parcelnumb = assessments.parcelnumb and assessments.taxyear = 2022
    join `cvilledata.cville_plans_together.flum_202110` flum on sales.parcelnumb = flum.parcelnumb
    where sales.saledate >= '2022-03-01' and sales.saledate < '2022-04-01'
      and sales.saleamount > 0
      and sales.bookpage not in (
        select
          bookpage
        from `cvilledata.cville_open_data.real_estate_sales`
        where saledate >= '2022-03-01' and saledate < '2022-04-01'
          and saleamount > 0
        group by bookpage
        having count(*) > 1
      )
      and flum.desig_oct in (
        'General Residential',
        'Medium-Intensity Residential',
        'High-Intensity Residential'
      )
    order by count desc
    """
).result().to_dataframe()

saleamount  assessment  saleassessmentratio  count  \
0    520000.0    412400.0             1.165224     43   
1    421000.0    396300.0             1.115945     14   

                    designation  
0           General Residential  
1  Medium-Intensity Residential

Looking at March alone, Medium-Intensity Residential parcels had sale prices exceeding their assessments by about 12%, and General Residential parcels had prices exceeding their assessmens by about 18%.

Let's revisit the claims from above:

> Parcels with the Medium-Intensity Residential designation have been selling for over their official assessments

This is consistent with the data! Medium-Intensity Residential parcels sold for about 5% more than assessments overall for 2022 and 12% in March alone. What about the next claim?

> The difference between sale prices and assessments is greater for Medium-Intensity Residential than for other designations

This claim isn't supported here. While Medium-Intensity Residential parcels sold for over their assessment prices, this was also true for General Residential parcels, and it turns out that sale prices exceeeded assessments by an even higher proportion in General Residential. Sean wrote that

> This month's key phrase is "Medium Intensity Residential".

But maybe the unusual thing about Medium-Intensity Residential is that the recent sale prices of these parcels have been lower, relative to assessments, than the General Residential category that makes up the bulk of the city's residential land.

I've tried to evaluate these claims in a bit more detail, but there are many limitations to this quick analysis. Maybe the Medium-Intensity Residential properties that have been sold this year are in different neighborhoods than the General Residential properties; maybe the city assesses properties differently by land use designation in a biased way. Maybe there was a pre-existing difference in land or property values across designations. Maybe correcting for these potential differences, or others that I haven't thought of, would support the claim that Medium-Intensity Residential properties are suddenly uniquely expensive. But at least so far, it appears that sale prices are higher than assessments citywide, and if anything this is more true of General Residential than Medium-Intensity Residential in recent months.

It's also worth pointing out that we don't have much data yet. The Comprehensive Plan was passed in December 2021, and only 23 parcels zoned Medium-Intensity Residential have sold in 2022 (not including multi-parcel transactions or homes built after 2020). Meanwhile, the zoning rewrite is just starting: we haven't actually changed the rules yet. It's interesting to look at the early data we have, but realistically, it's going to take years to draw conclusions with any confidence.

Anyway, thanks to Sean for raising some interesting questions. If you're still reading, subscribe to his [substack](https://communityengagement.substack.com/) if you haven't already.

One last point: maybe this analysis has been skewed by a small number of unusual sales. Using medians rather than means should reduce the influence of outliers. And I've excluded sales that I don't think we should consider: specifically, properties that sold for $0 and properties that were sold as a package along with other properties. But maybe I missed something! So for completeness, here are the individual parcels that have sold in 2022, first Medium-Intensity Residential, then General Residential:

In [4]:
#collapse-hide
bq.query(
    """
    select
      sales.parcelnumb as parcelnumber,
      sales.saleamount,
      sales.saledate,
      assessments.totalvalue,
      sales.saleamount / assessments.totalvalue as saleassessmentratio,
      sales.streetnumb,
      sales.streetname,
      sales.unit,
    from `cvilledata.cville_open_data.real_estate_sales` sales
    join `cvilledata.cville_open_data.real_estate_all_assessments` assessments on sales.parcelnumb = assessments.parcelnumb and assessments.taxyear = 2022
    join `cvilledata.cville_plans_together.flum_202110` flum on sales.parcelnumb = flum.parcelnumb
    where sales.saledate >= '2022-01-01'
      and sales.saleamount > 0
      and sales.bookpage not in (
        select
          bookpage
        from `cvilledata.cville_open_data.real_estate_sales`
        where saledate >= '2022-01-01'
          and saleamount > 0
        group by bookpage
        having count(*) > 1
      )
      and flum.desig_oct = 'Medium-Intensity Residential'
    order by sales.saledate desc
    """
).result().to_dataframe()

parcelnumber  saleamount    saledate  totalvalue  saleassessmentratio  \
0     470041000      525000  2022-04-04      386300             1.359047   
1     050013000      962500  2022-03-31      706300             1.362735   
2     070003180      835000  2022-03-31      681700             1.224879   
3     550049000      385000  2022-03-31      400900             0.960339   
4     43C041000      312500  2022-03-29      325200             0.960947   
5     21A131200      360700  2022-03-28      312500             1.154240   
6     440028230      511000  2022-03-24      328400             1.556029   
7     41A101100      900000  2022-03-24      698700             1.288106   
8     25A175000      365000  2022-03-21      338700             1.077650   
9     110076000      550000  2022-03-16      425900             1.291383   
10    600145100      606500  2022-03-15      579900             1.045870   
11    220195101      350000  2022-03-10      425300             0.822949   
12    540249000      260000  2022-03-04      275700             0.943054   
13    470045110      330000  2022-03-04      391700             0.842481   
14    110061100      457000  2022-03-02      316500             1.443918   
15    200100100      345000  2022-02-24      347500             0.992806   
16    43C026000      314000  2022-02-22      368400             0.852334   
17    610017110      239000  2022-02-17      175500             1.361823   
18    190034340      235000  2022-02-17      376300             0.624502   
19    270056000      233000  2022-02-14      285500             0.816112   
20    540246000      286000  2022-02-08      293700             0.973783   
21    200031000      392000  2022-02-02      360000             1.088889   
22    440018100      300000  2022-01-28      387600             0.773994   
23    290266024      345000  2022-01-13      324700             1.062519   

   streetnumb              streetname  unit  
0         607              WATSON AVE  None  
1         807                RUGBY RD  None  
2         111             ALDERMAN RD  None  
3        1338           CHESAPEAKE ST  None  
4        2239              BANBURY ST  None  
5         103             LONGWOOD DR     E  
6        1435             WESTWOOD RD  None  
7        1616  MEADOWBROOK HEIGHTS RD  None  
8         159            BROOKWOOD DR  None  
9         516               VALLEY RD  None  
10       1010             ELLIOTT AVE  None  
11       1526              CHERRY AVE  None  
12        213               MEADE AVE  None  
13        905             MARSHALL ST  None  
14        126              MAYWOOD LN  None  
15       2601      JEFFERSON PARK AVE  None  
16       2238              BANBURY ST  None  
17        959                RIVES ST  None  
18        201        OLD LYNCHBURG RD  None  
19        405             ELLIOTT AVE  None  
20        305               MEADE AVE  None  
21        107               HARRIS RD  None  
22       1439               RUGBY AVE  None  
23        210               BURNET ST  None

In [5]:
#collapse-hide
bq.query(
    """
    select
      sales.parcelnumb as parcelnumber,
      sales.saleamount,
      sales.saledate,
      assessments.totalvalue,
      sales.saleamount / assessments.totalvalue as saleassessmentratio,
      sales.streetnumb,
      sales.streetname,
      sales.unit,
    from `cvilledata.cville_open_data.real_estate_sales` sales
    join `cvilledata.cville_open_data.real_estate_all_assessments` assessments on sales.parcelnumb = assessments.parcelnumb and assessments.taxyear = 2022
    join `cvilledata.cville_plans_together.flum_202110` flum on sales.parcelnumb = flum.parcelnumb
    where sales.saledate >= '2022-01-01'
      and sales.saleamount > 0
      and sales.bookpage not in (
        select
          bookpage
        from `cvilledata.cville_open_data.real_estate_sales`
        where saledate >= '2022-01-01'
          and saleamount > 0
        group by bookpage
        having count(*) > 1
      )
      and flum.desig_oct = 'General Residential'
    order by sales.saledate desc
    """
).result().to_dataframe()

parcelnumber  saleamount    saledate  totalvalue  saleassessmentratio  \
0     230094200      360000  2022-04-07      372100             0.967482   
1     41A006000      750000  2022-04-07      553800             1.354280   
2     230090000       60000  2022-04-01      218200             0.274977   
3     380045000      625000  2022-04-01      504100             1.239833   
4     25A116101      550000  2022-04-01      471000             1.167728   
5     560114200      622431  2022-04-01      104500             5.956278   
6     290164000      751100  2022-03-31      495300             1.516455   
7     410043000      592000  2022-03-31      462400             1.280277   
8     050006000     1300000  2022-03-31     1154100             1.126419   
9     18A077B00      525000  2022-03-31      414600             1.266281   
10    490054000      400000  2022-03-31      375000             1.066667   
11    540064100      375000  2022-03-30      360000             1.041667   
12    48A040039      737995  2022-03-30      160000             4.612469   
13    41A076000      711501  2022-03-30      549900             1.293873   
14    260072000      265500  2022-03-28      239900             1.106711   
15    48A040015      706807  2022-03-25      160000             4.417544   
16    600088000      388000  2022-03-24      295100             1.314809   
17    200136000      485000  2022-03-24      418200             1.159732   
18    610065200      449000  2022-03-23      300600             1.493679   
19    600120007      520000  2022-03-23      429600             1.210428   
20    230048000      270000  2022-03-22      233600             1.155822   
21    18A013000      315000  2022-03-22      342100             0.920783   
22    310035000      645000  2022-03-21      563600             1.144429   
23    470048800      750000  2022-03-21      692200             1.083502   
24    48A040005      202860  2022-03-21      160000             1.267875   
25    48A040008      200655  2022-03-21      160000             1.254094   
26    48A040029      157658  2022-03-21      160000             0.985363   
27    060048000     1055000  2022-03-18     1059500             0.995753   
28    18A121000      605000  2022-03-18      487300             1.241535   
29    030026000     1150000  2022-03-18      789900             1.455880   
30    240071000      305000  2022-03-18      301400             1.011944   
31    400034000     1225000  2022-03-17     1051300             1.165224   
32    48A040038      806222  2022-03-17      160000             5.038888   
33    550066180      550000  2022-03-11      476700             1.153765   
34    520016400      599000  2022-03-10      590700             1.014051   
35    240068000      425000  2022-03-10      356600             1.191812   
36    200019100       65000  2022-03-09      150000             0.433333   
37    260043110      480000  2022-03-08      444300             1.080351   
38    070003600     1295000  2022-03-08      931000             1.390977   
39    610079150      236500  2022-03-08      243000             0.973251   
40    48A040034      739993  2022-03-07      160000             4.624956   
41    510162000      337000  2022-03-04      321700             1.047560   
42    42A087000      970000  2022-03-04      808200             1.200198   
43    47A095000      330000  2022-03-02      293400             1.124744   
44    21A023000      455000  2022-03-01      412400             1.103298   
45    190037290      380000  2022-03-01      347700             1.092896   
46    210080000      497000  2022-03-01      424300             1.171341   
47    480050000      520000  2022-03-01      428900             1.212404   
48    060068000     1375000  2022-03-01      919400             1.495541   
49    43B008000      522600  2022-02-28      430400             1.214219   
50    55A088618      468000  2022-02-28      390100             1.199692   
51    22A013000      435000  2022-02-25      522400